draft-ietf-ppm-dap.md

```diff
@@ -199,6 +222,10 @@ aggregator.
 - Clarify that differential privacy mitigations can help with privacy, rather
   than robustness, in the operational considerations section.
 
+- Rename "upload extension" to "report extension", and "prepare error" to
```

Comment:
This change should be noted for DAP-13, rather than DAP-12. The renaming PRs landed after DAP-12 was cut (see https://github.com/ietf-wg-ppm/draft-ietf-ppm-dap/commits/main/, relevant commits are 9541879 & a872ebe for the renamings, and ab7f534 for the DAP-12 tag.)

Reply:
How on earth did I do that.

Reply:
Done.